bundles/org.openhab.binding.vallox/src/main/resources/ESH-INF/thing/se-channels.xml

```diff
@@ -0,0 +1,502 @@
+	</channel-type>
+
+	<channel-type id="seFanSpeedMax">
+		<item-type>Number</item-type>
+		<label>Maximum Fan Speed</label>
+		<category>Fan</category>
+		<state min="1" max="8" step="1" pattern="%d" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seFanSpeedMin">
+		<item-type>Number</item-type>
+		<label>Minimum Fan Speed</label>
+		<category>Fan</category>
+		<state min="1" max="8" step="1" pattern="%d" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seTempInside">
+		<item-type>Number:Temperature</item-type>
+		<label>Inside Temperature</label>
+		<description>Current temperature inside the building.</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="seTempOutside">
+		<item-type>Number:Temperature</item-type>
+		<label>Outside Temperature</label>
+		<description>Current temperature outside the building.</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="seTempExhaust">
+		<item-type>Number:Temperature</item-type>
+		<label>Exhaust Temperature</label>
+		<description>Current temperature of the air flow exiting the building.</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="seTempIncoming">
+		<item-type>Number:Temperature</item-type>
+		<label>Incoming Temperature</label>
+		<description>Current temperature of the air flow incoming to the building.</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="sePowerState">
+		<item-type>Switch</item-type>
+		<label>Power State</label>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seCo2AdjustState" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Co2 Adjust State</label>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seHumidityAdjustState" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Humidity Adjust State</label>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="sePostHeatingState">
+		<item-type>Switch</item-type>
+		<label>Post Heating State</label>
+		<state readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seDcFanInputAdjustment" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Input Fan Speed</label>
+		<description>Adjust input fan speed in scale of 0-100%</description>
+		<state min="0" max="100" step="1" pattern="%d %%" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="seDcFanOutputAdjustment" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Output Fan Speed</label>
+		<description>Adjust output fan speed in scale of 0-100%</description>
+		<state min="0" max="1" pattern="%d %unit%" readOnly="false"></state>
```

Comment:
You pattern will only output whole number values, but you have set the max to `1`. Is it supposed to be `100`?

Reply:
Yes it is. Good catch.